script.js

```diff
@@ -0,0 +1,23 @@
+//JavaScript for Kids
+
+
+
+
+//Random insult generator 
+Math.floor //rounds up number
+Math.random //gives random number from 0-1
+
+
+const firstWord = ['Scheiß', 'Dumm' , 'Ray' , 'Arsch'];
+const secondWord = ['klo', 'kotz', 'fett', 'schakal'];
+let thridWord = ['idiot', 'kopf' , 'schwein' , 'pimmel'];
+
+thridWord.push = ('lauch'); //Why is it not adding it to the Array?
```

Comment:
because `push` is an array method (something like function) - therefore you have to execute it instead of assigning value to it

definition and examples https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push